.changeset/orange-roses-give.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
+---
+
+Changed ProgressBar.Item to require an aria-label.
```

Comment:
Would I be right in understanding that folks must provide an `aria-label` now? Otherwise, they'll see the warning showing up?

If so, could we make this a lint rule on `eslint-plugin-primer-react` instead and add updating the types for this component for our next major release batch? Just so that `aria-label` is required (unless `aria-hidden` is provided).

I think that making things required that previously weren't would fall under our `major` bumps but let me know if I'm misinterpreting anything 👀

Reply:
Good check -- I forgot to update this when I moved away from enforcing the prop and instead went to warning users 😓